scalableminds/webknossos
# Prefer annotation zoom over dataset zoom

This PR changes the initialization of tracings and so that the zoom level of annotations is preferred over the zoom set as default by the dataset. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- (just a guess though): Create an explorative tracing and change the zoom level. Now save the state of the tracing. Then remove the whole state (position, zoom and ...) from the url and reload the page. The zoom level should be the same as before.

### Issues:
- fixes #3962

------
